draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -558,6 +561,9 @@ delay product, well below the standard operating range of most Internet flows.
 
 Satellite links often have delays longer than typical for wired paths
 {{?RFC2488}} and high delay bandwidth products {{?RFC3649}}.
+Many systems
+use dynamic capacity assignment that can result in variation of the delay
+and the capacity over timescales of the order of the path RTT.
```

Comment:
Honest question: Is delay variation due to "dynamic capacity assignment" meaningfully different from queueing delay.

Reply:
That's a very old debate. On one hand, in CSMA systems, the mean time to wait before transmission is indeed linked to the network load. Increased delays are a sign of congestion. But that's not always the case. In mobile networks, a station may well move to a different cell. The delay may be longer, if the new base station is further away, or needs to poll more clients. The data rate may or may not be lower, depending of frequencies and bandwidth available. The old logic breaks.

Reply:
The case I was thinking of was Bandwidth on Demand rather than CSMA, where capacity requests are made to an allocator and capacity is assigned from a resource pool. The two main complications are: (1) The time to respond is not related to the traffic load: it is a function of the link RTT, and the allocation interval (often related to the super frame size in the systems I know of), but also dependent on the design of the method ... these have nothing to do with the queue occupancy (2) The available capacity from the resource pool depends on what can be used by the specific system ... which may be a subset of the total resource ... and also the load on that resource by other systems, propagation conditions, implemented waveforms, etc.

In short: when there is a queue, I see dynamic capacity assignment can cause delay, but the length of the delay is a function of an external control system, not the level of congestion for the flow - i.e. the delay can be transient and have no bearing on the availability of capacity next RTT. Whereas, for a FIFO queue, the queueing delay can have a strong correlation with the availability of capacity next RTT.

Reply:
So... let's see of what actually ought to be written for *evaluation* ?